cli/tests/test_help_command.rs

```diff
@@ -100,11 +102,11 @@ fn test_help() {
     ");
 
     let output = test_env.run_jj_in(".", ["help", "log", "--", "-r"]);
-    insta::assert_snapshot!(output, @r"
-    ------- stderr -------
-    Error: Unknown command: log -r
-    [EOF]
-    [exit status: 2]
+    let stdout = output.stdout.normalized();
```

Comment:
The help command has been made more forgiving, which I personally prefer. Not sure if others agree